src/examples/aes/aes_test.cpp

```diff
@@ -40,7 +40,9 @@ int32_t read_test_options(int32_t* argcp, char*** argvp, e_role* role, uint32_t*
 			{ (void*) &int_sharing, T_NUM, "g", "Sharing in which the AES circuit should be evaluated [0: BOOL, 1: YAO, 4: SP_LUT], default: BOOL", false, false },
 			{ (void*) verbose, T_FLAG, "v", "Do not print the result of the evaluation, default: off", false, false },
 			{ (void*) nthreads, T_NUM, "t", "Number of threads, default: 1", false, false },
-			{ (void*) use_vec_ands, T_FLAG, "u", "Use vector AND optimization for AES circuit for Bool sharing, default: off", false, false } };
+			{ (void*) use_vec_ands, T_FLAG, "u", "Use vector AND optimization for AES circuit for Bool sharing, default: off", false, false },
+			{ (void*) extend_in_sfe, T_FLAG, "x", "Calculate the key expansion during the SFE, default: false", false, false },
```

Comment:
Since its key *expansion*, maybe reflect this in the variable names, e.g. as `expand_in_sfe`.

Reply:
I was thinking hard about a good name, but I could not find a better than you suggested. I will change it.